home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/Permission.java

```diff
@@ -26,7 +27,7 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @ManyToMany(mappedBy = "permissions")
+    @ManyToMany(mappedBy = "permissions", fetch = FetchType.LAZY)
```

Comment:
Isn't lazy the default one?

Reply:
Yes, it is. I can remove it, if it's critical